driver/src/main/java/org/neo4j/driver/internal/async/pool/ExtendedChannelPool.java

```diff
@@ -20,9 +20,13 @@
 
 import io.netty.channel.pool.ChannelPool;
 
+import java.util.concurrent.CompletionStage;
+
 public interface ExtendedChannelPool extends ChannelPool
 {
     boolean isClosed();
 
     String id();
+
+    CompletionStage<Void> repeatableCloseAsync();
```

Comment:
Wdyt about calling this `destroyAsync`? I do not quite understand the "repeatable" part of the name :)